react/features/connection-indicator/components/web/ConnectionIndicatorContent.js

```diff
@@ -304,7 +309,8 @@ export function _mapStateToProps(state: Object, ownProps: Props) {
     const props = {
         _connectionStatus: participant?.connectionStatus,
         enableSaveLogs: state['features/base/config'].enableSaveLogs,
-        disableShowMoreStats: state['features/base/config'].disableShowMoreStats
+        disableShowMoreStats: state['features/base/config'].disableShowMoreStats,
```

Comment:
style: we add `_` for the props that are coming from `_mapStateToProps`. Could you please fix it?

Reply:
Could you please make sure all the props from this `_mapStateToProps` are following the convention with `_`?